src/ccnl-core/include/ccnl-interest.h

```diff
@@ -32,40 +32,94 @@
 #include "evtimer_msg.h"
 #endif
 
-struct ccnl_pendint_s { // pending interest
-    struct ccnl_pendint_s *next; // , *prev;
-    struct ccnl_face_s *face;
-    uint32_t last_used;
+/**
+ * ?
```

Comment:
This defines a flag. If set the ccn-lite forwarder should forward/retransmit a packet, otherwise it will be internally.